src/components/ExchangeLists.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { BsBoxArrowInRight } from 'react-icons/bs';
+// import bittrex from '../assets/bittrex.png';
+
+const ExchangeLists = ({ exchange, showDetails }) => {
+  const {
+    exchangeId, name, rank, volumeUsd, exchangeUrl,
+  } = exchange;
+
+  const money = new Intl.NumberFormat('en-US',
+    { style: 'currency', currency: 'USD' });
+
+  return (
+    <div className="flex flex-col text-white bg-blue-900 rounded-md hover:scale-105 hover:mx-3 hover:bg-blue-800">
+      <button
+        type="button"
+        aria-label="click"
+        onClick={() => showDetails(exchangeId)}
+        className=" self-end p-4 hover:text-[#ff7daf]"
```

Comment:
[OPTIONAL]
- [ ] For a better user experience, It would be nicer to show the detail of a specific coin when the user clicks the individual category item and not the arrow.

Reply:
Thank you... I am going to implement that now...